packages/data/src/world/mq/deku_tree_mq.yml

```diff
@@ -2,126 +2,282 @@
+  events:
+    DEKU_MQ_ROOM_AFTER_WATER_CLEAR: "soul_keese && soul_deku_scrub && soul_enemy(SOUL_ENEMY_GOHMA_LARVA) && (has_weapon || has_ranged_weapon_child)" #Judgment call, this fight is way too big for sticks and nuts to be forced.
+  locations:
+    "MQ Deku Tree Grass Larvae Room 1": "can_cut_grass"
+    "MQ Deku Tree Grass Larvae Room 2": "can_cut_grass"
+"MQ Deku Tree Room After Water Room Door to Gravestone Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Room After Water Room": "true"
+    "MQ Deku Tree Gravestone Room Door to Room After Water Room": "event(DEKU_MQ_ROOM_AFTER_WATER_CLEAR)"
+"MQ Deku Tree Gravestone Room Door to Room After Water Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Room After Water Room Door to Gravestone Room": "true"
+    "MQ Deku Tree MQ Grave Room": "true"
+"MQ Deku Tree MQ Grave Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Gravestone Room Door to Room After Water Room": "true"
+    "MQ Deku Tree Back Side Room Entry Door": "can_use_sticks || has_fire"
+    "MQ Deku Tree Basement Ledge": "is_child && (can_use_sticks || can_use_din)"
+  events:
+    STICKS: "can_kill_baba_sticks"
+    NUTS: "can_kill_baba_nuts"
   locations:
     "MQ Deku Tree GS Song of Time Blocks": "gs && ((can_play_time && can_collect_distance) || can_longshot)"
-    "MQ Deku Tree GS Back Room": "gs && (can_use_sticks || has_fire) && can_collect_distance"
     "MQ Deku Tree Grass Gravestone Room 1": "can_cut_grass"
     "MQ Deku Tree Grass Gravestone Room 2": "can_cut_grass"
     "MQ Deku Tree Grass Gravestone Room 3": "can_cut_grass"
     "MQ Deku Tree Grass Gravestone Room 4": "can_cut_grass"
     "MQ Deku Tree Grass Gravestone Room 5": "can_cut_grass"
-    "MQ Deku Tree Grass Back Room 1": "can_cut_grass && (can_use_sticks || has_fire)"
-    "MQ Deku Tree Grass Back Room 2": "can_cut_grass && (can_use_sticks || has_fire)"
-    "MQ Deku Tree Grass Back Room 3": "can_cut_grass && (can_use_sticks || has_fire)"
-    "MQ Deku Tree Grass Larvae Room 1": "can_cut_grass" #You can go backwards to get this specific grass with nothing else.
-    "MQ Deku Tree Grass Larvae Room 2": "can_cut_grass" #You can go backwards to get this specific grass with nothing else.
     "MQ Deku Tree Wonder Item 1": "can_use_sword_or_sticks"
     "MQ Deku Tree Wonder Item 2": "can_use_sword_or_sticks"
     "MQ Deku Tree Wonder Item 3": "can_use_sword_or_sticks"
     "MQ Deku Tree Wonder Item 4": "can_use_sword_or_sticks"
-"Deku Tree Basement Ledge":
+"MQ Deku Tree Back Side Room Entry Door":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree MQ Grave Room": "true"
+    "MQ Deku Tree Back Side Room Exit Door": "true"
+"MQ Deku Tree Back Side Room Exit Door":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Back Side Room Entry Door": "true"
+    "MQ Deku Tree Back Side Room": "true"
+"MQ Deku Tree Back Side Room":
+  dungeon: DT
+  exits:
+    "MQ Deku Tree Back Side Room Exit Door": "true" #One-sided collision for webbing; might need to be changed for door rando.
```

Comment:
If the web is visible, it needs to use the trick. Otherwise, this is fine. (Same for the other webs in the main room if you spawn behind them)

Reply:
I believe it is the same webbing as the Back Room; but that's why I put in the comment.